### DISCUSSION TITLE
version 2.8 phase display The new tool was not found

### DISCUSSION THREAD

**Discussion by Terracotta-62** [artisan-roaster-scope/artisan#992] [General]
![20221022113558](https://user-images.githubusercontent.com/116416332/197317410-44b22d25-729a-443a-9717-7af7ce6ed39a.png)

**Comment by MAKOMO** [comment]
??

**Comment by roasterdave** [comment]
Earlier on in the **[webinar](https://www.youtube.com/watch?v=tJKRJrEPeEQ)** I explained the need to drag up from the bottom of the graph to show the phases.  There is light grey bar across the bottom where the mouse will change to a drag cursor.  This animation, taken from the **[release notes blog post](https://artisan-roasterscope.blogspot.com/2022/10/artisan-v280.html)**, should help.


![image](https://blogger.googleusercontent.com/img/b/R29vZ2xl/AVvXsEjmrb43xtiH9UJxwm4ZBZH0zHX_Kxx_OtxpDnPQ5bmgCGCkz7Tu6ILDK-4e6sUUD_epdhTQ3cErqOHPq-OQ5VBARScuQ_SU8fNRQaOcfkANCimDj9BcPx9TwFQlKP6jUtmapy4w-6uHGSpHyoNw5SuLX4AxfPDuow5wKpdgwKMr_fl3Bex95SSO1okL/s900/comparator-widget.mov.gif?imgmax=1600)

**Comment by Terracotta-62** [comment]
Thank you for the demo. We got it ！
